FriendsListTentative

```diff
@@ -0,0 +1,98 @@
+/****************************
+ * Friend List
+ * Original idea from WGC
+ * Coded by Volco
```

Comment:
On the topic of “do not try and pass other people’s code as your own.” Is this yours? I swear this is Phil’s or JD’s code, no?

Reply:
It is not mine. For some dumb reason I thought it wouldn't matter whether I included the author of the code or not. I definitely know better now.

Reply:
No austin, i made this for WL. Jd's is probably still on wisps repo if he has it still and i don't even remember when phils is.
Compare it if you want but i did make it from scratch

Reply:
Fair enough, it does seem to be heavily based off jd’s but you’ve done enough to warrant it to be your own

Reply:
I didn't even look at jd's at all tho lol, but w/e